articles/multifactor-authentication/duo/user-guide.md

```diff
@@ -0,0 +1,40 @@
+# How to Use Duo
```

Comment:
Doesn't the concepts regarding user name and password belong to Auth0 lock or such kind of widget? Do we want to state this info here? I mean user can customize the login page, it can even get rid of lock completely, so the info might not always be true (but i'd be most of the time)